docs/source/platform/schema-validation.md

```diff
@@ -147,7 +146,31 @@ After analyzing the changes against current usage metrics, Apollo will identify
 2. **Warning**: There are potential problems that may come from this change, but no clients are immediately impacted.
 3. **Notice**: This change is safe and will not break current clients.
 
-The more [performance metrics](./performance.html) that Apollo has, the better the report of these changes will become.
+<h3 id="cli-advanced">Advanced CLI Usage</h3>
+
+By default, the CLI will look at operations run within the last day in order to decide which changes are breaking. This period of time may not be large enough, so the `validationPeriod` flag accommodates custom timeframes:
+
+```bash
+apollo service:check --validationPeriod=P2W
+```
+
+> Valid durations are represented in ISO 8601. For reference, see: https://bit.ly/2DEJ3UN. It can also be provided as a number in seconds, i.e. 86400 for a single day.
```

Comment:
Could we put the link in one of these [things](www.things.things.things)?

Reply:
Resovled in 118f491f5b67266c2f89f7da62d65d4b52bd5983